src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,47 @@
+package mate.academy.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import java.time.LocalDateTime;
+
+@Entity
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    private Movie movie;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
```

Comment:
Using `CascadeType.PERSIST` on `@ManyToOne` associations may not always be desired. It will persist the associated `Movie` or `CinemaHall` when you persist a `MovieSession`. Make sure this behavior is intended as per your application's requirements. If not, you might want to remove the cascade option or choose a different cascade type.